### ISSUE TITLE
`preResolveTypes` refers to wrong type (missing array)

### ISSUE THREAD

**Issue by SimenB** [dotansimha/graphql-code-generator#2159]
**Describe the bug**
 

When using the new `preResolveTypes` config option, the generated types lose their Array in fragments.

**To Reproduce**

https://codesandbox.io/embed/graphql-codegen-preresolvetypes-bug-5c821

**Expected behavior**

```ts
export type ThingItemFragment = {
  __typename?: 'Thing';
  requirements: { __typename?: 'Requirements'; unmetRequirements: Array<Requirement> };
};
```

**Environment:**

- OS: macOS
- `@graphql-codegen/...`: 1.4.0
- NodeJS: 10.16.0

**Additional context**

**Comment by dotansimha** [comment]
Fixed in https://github.com/dotansimha/graphql-code-generator/pull/2270

**Comment by dotansimha** [comment]
Fixed in 1.5.0 🎉
